test/fuzzing/build.sh

```diff
@@ -0,0 +1,28 @@
+export FUZZER_DIR=$SRC/linkerd2/test/fuzzing
+
+mkdir $SRC/linkerd2/test/fuzzing
```

Comment:
You probably want the `-p` flag to prevent `mkdir` from failing if the directory already exists (if it should fail there probably needs to be some cleanup function added).

```suggestion
mkdir -p "$SRC"/linkerd2/test/fuzzing
```